pkg/daemons/control/server.go

```diff
@@ -323,6 +331,10 @@ func cloudControllerManager(ctx context.Context, cfg *config.Control) error {
 		argsMap["controllers"] = argsMap["controllers"] + ",-cloud-node,-cloud-node-lifecycle"
 		argsMap["secure-port"] = "0"
 	}
+	if cfg.MultiClusterCIDR {
+		argsMap["cidr-allocator-type"] = "MultiCIDRRangeAllocator"
+		argsMap["feature-gates"] = "MultiCIDRRangeAllocator=true"
```

Comment:
use this: https://github.com/k3s-io/k3s/blob/7739c8b97ee453a82d2762f2c2172c64e4aaa825/pkg/util/gates.go#L3-L4